internal/factsengine/gatherers/corosynccmapctl.go

```diff
@@ -0,0 +1,45 @@
+package gatherers
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	CorosyncCmapCtlFactKey = "corosync-cmapctl"
+)
+
+type CorosyncCmapctlGatherer struct {
+	executor CommandExecutor
+}
+
+func NewCorosyncCmapctlGatherer() *CorosyncCmapctlGatherer {
+	return &CorosyncCmapctlGatherer{
+		executor: Executor{},
+	}
+}
+
+func (s *CorosyncCmapctlGatherer) Gather(factsRequests []FactRequest) ([]Fact, error) {
+	facts := []Fact{}
+	log.Infof("Starting corosync-cmapctl facts gathering process")
+
+	corosyncCmapctl, err := s.executor.Exec( // nolint:gosec
```

Comment:
Why are we disabling linting here? I think we can remove it safely as the arguments are constants